install/Knative-with-GKE.md

```diff
@@ -194,7 +194,7 @@ for getting started:
   deploy one of your choosing.
 
 * To get started by creating a build, see
-  (Creating a simple Knative Build)[../build/creating-builds.md]
+  [Creating a simple Knative Build](../build/creating-builds.md)
```

Comment:
This change fixes the formatting, but the linked page doesn't exist. @RichieEscarez , where did you want this link to go?

Reply:
My guess is that this file is still in the making, see this PR https://github.com/knative/docs/pull/261